doc/source/deployment/setup-deployer.rst

```diff
@@ -0,0 +1,179 @@
+     deploy_airship [label="Deploy Airship"]
+     deploy_openstack [label="Deploy OpenStack"]
+
+     group {
+       deployer
+       color="red"
+     }
+
+     deployer -> ses_integration;
+     ses_integration -> configure;
+     configure -> setup_caasp_workers;
+
+     group {
+       color = "#EEEEEE"
+       label = "Cloud Deployment"
+       setup_caasp_workers -> patch_upstream;
+       patch_upstream -> build_images;
+       build_images -> deploy_airship [folded];
+       setup_caasp_workers -> deploy_airship;
+       deploy_airship -> deploy_openstack;
+     }
+   }
+
+
+Base Software
+-------------
+
+The following software must be installed on your `Deployer`:
+
+  * ansible>=2.7.0
+  * gcc
+  * git
+  * jq
+  * python3-netaddr
+  * python-virtualenv
+
+Create SUSE Containerized OpenStack Workspace
+---------------------------------------------
+
+All the deployment artifacts are stored in a :term:`workspace`. The workspace
+by default is a directory located in the user's home directory on the Deployer.
+Create a directory in your home directory that ends in -workspace. Then export
+SOCOK8S_ENVNAME=<directory name prefix> to set your workspace. To change your
+workspace parent directory, export `SOCOK8S_WORKSPACE_BASEDIR` with the base
+directory where your workspace is located.
+
+.. code-block:: console
+
+  mkdir ~/socok8s-workspace
+  export SOCOK8S_ENVNAME=socok8s
+
+
+Cloning repository
+-----------------------
+
+To get started, clone the socok8s GitHub repository. This repository uses
+submodules, so you need to get all the code to make sure the playbooks work.
+
+::
+
+   git clone --recursive https://github.com/SUSE-Cloud/socok8s.git
+
+Alternatively, one can fetch/update the tree of the submodules by running:
+
+::
+
+   git submodule update --init --recursive
+
+
+SSH Key Preparation
+-------------------
+
+Create an SSH key on the Deployer node, and add the public key to each CaaS
+Platform worker node.
+
+.. note ::
+
+  1. To generate the key, you can use ssh-keygen -t rsa
+
+  2. To copy the ssh key to each node, use the ssh-copy-id command,
+     for example: ssh-copy-id root@192.168.122.1
+
+  Test this by connecting to the node via SSH and executing a command with ‘sudo’.
+  Neither operation should require a password.
+
+Passwordless sudo
+-----------------
+
+If installing as a non-root user, you will need to give your user passwordless
+sudo on the Deployer.
+
+.. code-block:: console
+
+   sudo visudo
+
+Add the following.
+
+.. code-block:: console
+
+   <username> ALL=(ALL) NOPASSWD: ALL
+
+Add the above line after "#includedir /etc/sudoers.d". replace <username> with
+your username.
+
+
+Configure Ansible
+-----------------
+
+Use ARA (recommended)
+~~~~~~~~~~~~~~~~~~~~~
+
+Ansible Run Analysis (ARA) makes Ansible runs easier to visualize, understand,
+and troubleshoot. To use ARA, set the following environment variable before
+running `run.sh`.
```

Comment:
dont we need to install ara first for this to work? especially if using package install. BTW I dont think we provide python-ara as package yet